server/src/main/java/io/spine/server/route/MessageRouting.java

```diff
@@ -74,6 +74,16 @@
         return this;
     }
 
+    /**
+     * Verifies if the passed message type is supported by this instance of routing.
```

Comment:
```suggestion
     * Checks if the passed message type is supported by this instance of routing.
```

For me, _verify_ is closer to _validate_.